### DISCUSSION TITLE
Customer Token Access

### DISCUSSION THREAD

**Discussion by blueportal31415** [saleor/saleor#5992] [Q&A] [answered]
Does anyone know why customer information cant be queried with a customer token? I can query customers using user(id: $id) with an admin token, but cant do it with a customer token

**Comment by patrys** [comment, marked as answer]
If you want to get the currently logged in user's data, you can use the `me` field of the root `Query` type:

```graphql
query Example {
  me {
    email
  }
}
```

**Reply by blueportal31415** [reply]
thank you so much!

**Reply by blueportal31415** [reply]
@patrys I appreciate your help! 

Do you or anyone you know that is very familiar with Saleor is open to do freelance work? 
If so please get in touch at info@blueportal.io
